commit 00578c3036e0a267cd50e63d3745264a533d1899
Author: Christoph Feck <christoph@maxiom.de>
Date:   Mon Aug 13 16:41:37 2012 +0200

    Use forward declaration
    
    Instead of including a file from another target, which might
    not have been generated yet.
    
    BUG: 304987
    FIXED-IN: 4.9.1
    (cherry picked from commit 46b9acb42eed48f0fd3b9d2391dc8a217003cafc)

diff --git a/kcontrol/randr/randrconfig.h b/kcontrol/randr/randrconfig.h
index 37f7b57..86fdefc 100644
--- a/kcontrol/randr/randrconfig.h
+++ b/kcontrol/randr/randrconfig.h
@@ -22,7 +22,6 @@
 
 #include "ui_randrconfigbase.h"
 #include "randr.h"
-#include "outputconfig.h"
 
 #include <QWidget>
 #include <QTimer>
@@ -33,6 +32,9 @@ class CollapsibleWidget;
 class RandRDisplay;
 class OutputGraphicsItem;
 class LayoutManager;
+class OutputConfig;
+
+typedef QList<OutputConfig*> OutputConfigList;
 
 class RandRConfig : public QWidget, public Ui::RandRConfigBase
 {
